### PR TITLE
Fix struct names in comment

### PR DESCRIPTION
**Pull request by hishope**
### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

**Review by rtribotte — APPROVED** [review]
Thanks 👍

File: pkg/provider/docker/data.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ type networkSettings struct {
 	Networks    map[string]*networkData
 }
 
-// Network holds the network data to the provider.
+// NetworkData holds the network data to the provider.
```
Comment (kevinpollet) [inline pkg/provider/docker/data.go:28]:
```suggestion
// networkData holds the network data to the provider.
```

**Review by kevinpollet — CHANGES_REQUESTED** [review]

**Comment by kevinpollet** [comment]
Hello @hishope,

As this pull request is a fix, could you please rebase it on branch v2.11?

**Comment by mmatur** [comment]
> Hello @hishope,
> 
> As this pull request is a fix, could you please rebase it on branch v2.11?

This code has been introduced in Traefik v3.0, the PR has been rebase on v3.0 accordingly

**Review by mmatur — APPROVED** [review]

**Pull request merged**